### DISCUSSION TITLE
Help needed

### DISCUSSION THREAD

**Discussion by matt2005** [opencardev/crankshaft#693] [General]
I'm really busy with work and Family and am looking for some help maintaining.

**Comment by SonOfGib** [comment]
I am interested in helping. I am able to build and test openauto/aasdk and I have a few different RPIs sitting around I've been using.

I also have a working (if basic) crankshaft installation that gets daily use.

**Reply by matt2005** [reply]
I've added you to the dev team.

**Reply by mi-hol** [reply]
Hi, would like to help with project administration tasks like issue mgmt (i.e. going thru issue backlog and flowing up/closing https://github.com/opencardev/crankshaft/issues/557) and automating related workflows
